examples/mobile/UIComponents/components/appbar/navigateUpButton.html

```diff
@@ -13,15 +13,15 @@
 	<div class="ui-page" id="simpleTitleBack">
 		<header>
 			<div class="ui-appbar-left-icons-container">
-				<button class="ui-btn ui-btn-icon ui-btn-icon-back" data-rel="back"></button>
+				<button class="ui-btn ui-btn-icon ui-btn-icon-back" data-style="flat" data-rel="back"></button>
```

Comment:
You should change also backgorund-color for button::before pseudo element
background-color: transparent;

Reply:
Ok, added.